tests/manage/noobaa/test_write_to_bucket.py

```diff
@@ -0,0 +1,62 @@
+import logging
+
+import boto3
+import pytest
+
+from ocs_ci.framework import config
+from ocs_ci.framework.pytest_customization.marks import tier1
+from ocs_ci.ocs import constants
+from tests.helpers import create_unique_resource_name
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.skipif(condition=True, reason="NooBaa is not deployed")
+@pytest.mark.skipif(
+    condition=config.ENV_DATA['platform'] != 'AWS',
+    reason="Tests are not running on AWS deployed cluster"
+)
+@tier1
+class TestBucketIO:
```

Comment:
Do we want to inherit from our [base class](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/testlib.py) / create a specific one for NooBaa testing? As far as I know we use those base classes to ensure the `run_io_in_background` and `environment_checker` fixtures are executed for the test session.

Reply:
Inheriting from the base class should suffice. Thank you :)